manifests/pulp.pp

```diff
@@ -129,50 +134,81 @@
     }
   }
 
+  if $expose_mongodb {
+    include certs::mongodb
+    include certs::mongodb_client
```

Comment:
I think this should be unconditional now: we always need the client certs so moved out of this if block.

Reply:
We only need the mongodb client certificates if we are exposing mongodb, otherwise they go unused.

Reply:
Right. And we'll also need it in foreman_proxy_content with pulpcore migrator plugin.